### PR TITLE
chore: bump deps + move to jsr

### PR DESCRIPTION
**Pull request by load1n9**

### CODE REVIEW

File: rs_lib/src/lib.rs [hunk 1]
```diff
@@ -27,7 +27,7 @@ pub fn generate_bindgen(
 fn inner(name: &str, wasm_bytes: Vec<u8>) -> Result<Output> {
   let mut x = wasm_bindgen_cli_support::Bindgen::new()
     .deno(true)?
-    .weak_refs(true)
+    // .reference_types(true)
```
Comment (iuioiua) [inline rs_lib/src/lib.rs:30]: Was this accidentally left in here?

Comment (littledivy) [reply rs_lib/src/lib.rs:30]:
```suggestion
```

weak refs is auto detected now. https://github.com/rustwasm/wasm-bindgen/commit/3e469b265464e60645f7eceb5745fa2558aea703

**Review by littledivy — APPROVED** [review]
LGTM

File: lib/deps.ts [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
-export * as base64 from "https://deno.land/std@0.198.0/encoding/base64.ts";
-export { parse as parseFlags } from "https://deno.land/std@0.198.0/flags/mod.ts";
-export { Sha1 } from "https://deno.land/std@0.142.0/hash/sha1.ts";
-export * as path from "https://deno.land/std@0.198.0/path/mod.ts";
-export { copy } from "https://deno.land/std@0.198.0/streams/copy.ts";
-export { writeAll } from "https://deno.land/std@0.198.0/streams/write_all.ts";
+export * as colors from "jsr:@std/fmt@0.215/colors";
+export * as base64 from "jsr:@std/encoding@0.215/base64";
+export { parseArgs as parseFlags } from "jsr:@std/cli@0.215/parse_args";
+export { Sha1 } from "./utils/sha1.ts";
```
Comment (littledivy) [inline lib/deps.ts:6]:
```suggestion
// TODO(@littledivy): use webcrypto digest API
export { Sha1 } from "./utils/sha1.ts";
```

**Comment by dsherret** [comment]
Before publishing we'll have to do this otherwise it won't work: https://github.com/denoland/wasmbuild/issues/121

**Pull request merged**